.travis.yml

```diff
@@ -5,11 +5,13 @@ python:
   - 3.6
 install:
   - pip install flake8
+  - npm install -g jshint
 before_script:
   # stop the build if there are Python syntax errors, PEP8 violations, undefined names
   - flake8 . --count --select=E,F821,F822,F823 --max-line-length=127 --show-source --statistics
   # exit-zero treats all errors as warnings.  GitHub editor is 127 chars wide
   - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+  - jshint . || true  # treats all errors as warnings
```

Comment:
I would prefer to have the build failing rather than pass with warning so I would remove the || true part